requirements.txt

```diff
@@ -1,8 +1,12 @@
-Flask==2.2.2
+Flask==2.1.3
```

Comment:
Why did you downgrade the version?

Reply:
Because there is a "Bug" in the werkzug (a curl-like library that is used by sentry and flask) that will not enable us to send requests unless you specify content-type, unfortunatly this safeguard is breaking sentry. No security issues between those two versions